R/render-table.R

```diff
@@ -3,33 +3,103 @@
 #' Creates a reactive table that is suitable for assigning to an \code{output}
 #' slot.
 #'
-#' The corresponding HTML output tag should be \code{div} and have the CSS class
-#' name \code{shiny-html-output}.
+#' The corresponding HTML output tag should be \code{div} and have the CSS
+#' class name \code{shiny-html-output}.
 #'
 #' @param expr An expression that returns an R object that can be used with
 #'   \code{\link[xtable]{xtable}}.
-#' @param ... Arguments to be passed through to \code{\link[xtable]{xtable}} and
-#'   \code{\link[xtable]{print.xtable}}.
+#' @param striped,condensed,hover,bordered Logicals: if \code{TRUE}, apply
+#'   the corresponding Bootstrap table format to the output table.
+#' @param width A string with the width of the table, expressed in any valid
+#'   CSS unit of length.
+#' @param align A string that specifies the column alignment. If equal to
+#'   \code{'l'}, \code{'c'} or \code{'r'}, then all columns will be,
+#'   respectively, left-, center- or right-aligned. Otherwise, \code{align}
+#'   must have the same number of characters as the resulting table (if
+#'   \code{rownames = TRUE}, this will be equal to \code{ncol()+1}), with
+#'   the \emph{i}-th character specifying the alignment for the
+#'   \emph{i}-th column (again, only \code{'l'}, \code{'c'} or
+#'   \code{'r'} are permitted). If \code{NULL}, then all numeric/integer
+#'   columns (including the row names, if they are numbers) will be
+#'   right-aligned and everything else will be left-aligned.
```

Comment:
Did you consider taking a vector of characters instead of a single string with col-per-letter? This feels less idiomatic R to me than that, though I like that it's concise. (I think @hadley does something similar in readr for col types IIRC) Just wondering what your thought process was.